qemu/tests/qemu_guest_agent.py

```diff
@@ -456,6 +490,48 @@ def gagent_check_sync(self, test, params, env):
         error_context.context("Check guest agent command 'guest-sync'", logging.info)
         self.gagent.sync()
 
+    @error_context.context_aware
+    def gagent_check_container(self, test, params, env):
+        """
+        Start qga container behavior only for linux guest
+        and release version above rhel8.2.0 now:
+        steps:
+        1) Prepare for container that stop qga.service and
+         install podman.
+        2) Create qga container.
+        3) start qga container.
+        4) Install target qga pkg we aim to test.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environmen.
+        """
+
+        session = self._get_session(params, None)
+        self._open_session_list.append(session)
+
+        error_context.context("Prepare env for qga container.", logging.info)
+        self.gagent_stop(session, self.vm)
+        self.podman_install(session, self.vm)
+
+        error_context.context("Create a qemu-guest-agent container.", logging.info)
+        cmd_create_container = params["create_container_cmd"]
+        s, o = session.cmd_status_output(cmd_create_container, timeout=120)
+        if s and "ERROR" in o:
```

Comment:
Recommend `or` instead of `and`, if it succeed any of the cases should not appear.

Reply:
okay,thanks.

Reply:
What do you think about this one?